tests/playbooks/test_minimal.yaml

```diff
@@ -20,6 +20,10 @@
   hosts: local
   gather_facts: false
   force_handlers: true
+  vars:
+    # @matbu: Should overrided by Zuul and be removed from here
```

Comment:
The test suite should be easily executable from a lab machine or a laptop that is connecting to a lab machine. Rather than using absolute paths, we can use paths relative to the playbook being run, like:

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/0ea5b7198572f74c12a6b4695e270c40f6c59f5e/tests/roles/mariadb_copy/defaults/main.yaml